README.md

```diff
@@ -58,144 +59,118 @@ The tomcat module requires [puppetlabs-stdlib](https://forge.puppetlabs.com/pupp
 puppet module upgrade puppetlabs-stdlib
 ~~~
 
-###Beginning with tomcat
+### Beginning with tomcat
 
-The simplest way to get Tomcat up and running with the tomcat module is to install the Tomcat package from EPEL...
+The simplest way to get Tomcat up and running with the tomcat module is to install the Tomcat source and start the service:
 
-~~~
-class { 'tomcat':
-  install_from_source => false,
+```puppet
```

Comment:
Hmmm. Isn't the typical "install and configure" module workflow to declare the class, and the class installs the software and configures it? Then multiple instances are managed via distinct instance resources? It seems weird to me that we'd reverse that here and declare a `tomcat::install` resource, which in turn includes a `tomcat` class.